content/blog/2021/06/2021-06-17-libera-chat.adoc

```diff
@@ -0,0 +1,27 @@
+---
+:layout: post
+:title: "Jenkins IRC moved to Libera Chat"
+:tags:
+- community
+- announcement
+- chat
+:author: markewaite
+links:
+  discourse: https://community.jenkins.io/t/jenkins-irc-moved-to-libera-chat/146
+opengraph:
+  image: /images/post-images/2021-06-libera-chat/jenkins-irc-on-libera-chat-opengraph.png
+---
+
+image:/images/post-images/2021-06-libera-chat/jenkins-irc-on-libera-chat-opengraph.png[Jenkins IRC now on Libera Chat, role=center, link="https://groups.google.com/g/jenkinsci-dev/c/pEcun5ZSfgo/m/ztRg03VNAgAJ"]
+
+The link:/chat/#internet-relay-chat-irc[Jenkins IRC channels] have moved to Libera chat.
+Many Jenkins developers have been dropped without warning from the previous Jenkins IRC chat system on Freenode.
+Rather than remain in a system that was increasingly unfriendly to our developers and administrators, the Jenkins community has moved to link:https://libera.chat/[Libera Chat IRC].
+
+A more detailed history of the transition is available on the link:https://groups.google.com/g/jenkinsci-dev/c/pEcun5ZSfgo/m/ztRg03VNAgAJ[Jenkins developers mailing list].
+Special thanks to Tim Jacomb, Gavin Mogan, Alex Earl, and Olivier Vernin for their work on the migration of IRC channels and automation.
+
+Other Jenkins chat channels like link:/chat/#gitter[Gitter] and link:/chat/#slack[Slack] are unchanged.
+We intend to replace many of those channels with link:https://community.jenkins.io/[community.jenkins.io].
```

Comment:
Soon? Over the long term? Do we have an expected time frame for this?

Reply:
```suggestion
We intend to gradually replace many of those channels with link:https://community.jenkins.io/[community.jenkins.io].
```

Works for you @daniel-beck ?